plugins/filetrees/chadtree.nix

```diff
@@ -0,0 +1,444 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.chadtree;
+  helpers = import ../helpers.nix {inherit lib;};
+  mkListStr = helpers.defaultNullOpts.mkNullable (types.listOf types.str);
+in {
+  options.plugins.chadtree = with helpers;
```

Comment:
usually we don't set `with helpers` and we explicitly write `helpers.mkBool` ...
Honestly I don't really mind if you do it. However, if you choose to do so, remove the useless mentions of `helpers.` bellow.

Reply:
I removed the `with helpers` doesn't make sense not to follow convention here.